pkg/curatedpackages/packagecontrollerclient.go

```diff
@@ -78,12 +94,12 @@ func (pc *PackageControllerClient) InstallController(ctx context.Context) error
 	return nil
 }
 
-func (pc *PackageControllerClient) ValidateControllerDoesNotExist(ctx context.Context) error {
+func (pc *PackageControllerClient) IsInstalled(ctx context.Context) (bool, error) {
 	found, _ := pc.kubectl.GetResource(ctx, "packageBundleController", packagesv1.PackageBundleControllerName, pc.kubeConfig, constants.EksaPackagesName)
 	if found {
-		return errors.New("curated Packages controller exists in the current cluster")
+		return true, errors.New("curated Packages controller exists in the current cluster")
 	}
-	return nil
+	return false, nil
 }
```

Comment:
Ah interesting
I was thinking something like

```go
func (pc *PackageControllerClient) IsInstalled(ctx context.Context) (bool, error) {
    found, err := pc.kubectl.GetResource(ctx, "packageBundleController", packagesv1.PackageBundleControllerName, pc.kubeConfig, constants.EksaPackagesName)
    if err != nil {
        return false, err
    }

    return found, nil
}
```

Reply:
Ditto. This ties in with my [other comment](https://github.com/aws/eks-anywhere/pull/3244#discussion_r961932500).

Reply:
Ok I think there is some confusion here. IsInstalled should lead to an error due to the fact that there is a controller that is already installed in the cluster. We wouldn't want to install the controller if there is already an installation. I hope this clears this up.